src/index.ts

```diff
@@ -135,7 +135,7 @@ export {
 export { ChromeStorageEngine } from './chrome-storage-engine';
 
 // Instantiate the configuration store with memory-only implementation.
-const configurationStore = configurationStorageFactory({
+const flagConfigurationStore = configurationStorageFactory({
```

Comment:
The underlying client now has multiple configuration stores, so naming this `flagConfigurationStore` so its more clear